experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/schemaBuilder.ts

```diff
@@ -0,0 +1,127 @@
+/*!
```

Comment:
I think it would make more sense to not depend on schema builder from editiable-tree-2, and instead just define a builder for create functions here, then use it from the domains/schemaBuilder.

Reply:
Wouldn't that create a dependency cycle where domains depends on editable-tree-2 and editable-tree-2 depends on domains? I struggled to find a way to make this not cyclical and I agree the result is weird. Maybe it would get better if `proxies/` was moved outside of `editable-tree-2`?

Reply:
I was able to do it as you suggested, and it's better. Thanks for the suggestion!